addon/components/au-fieldset/legend.hbs

```diff
@@ -0,0 +1,28 @@
+{{#if (has-block)}}
+<div ...attributes>
+  <legend class="au-c-fieldset__legend {{this.skin}} {{this.error}} {{this.warning}}">
+    {{#if this.warning}}
+    <AuBadge @icon="alert-triangle" @size="small" @skin="warning" class="au-u-margin-right-tiny" />
+    {{/if}}
+    {{#if this.error}}
+    <AuBadge @icon="cross" @size="small" @skin="error" class="au-u-margin-right-tiny" />
+    {{/if}}
+    {{yield "legend"}}
+    {{#if @required}}
+      {{#if @inline}}
+        {{#if @requiredLabel}}
+        <span class="au-c-fieldset__legend-addendum">{{@requiredLabel}}</span>
```

Comment:
nitpick, I think it would be "better" to use `au-c-fieldset-legend` and `au-c-fieldset-legend__addendum` instead?